graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -49,16 +48,21 @@
     HUGGINGFACE_TOKENIZE,
     TRANSFORMATIONS_FILENAME,
 )
-from ..config.config_parser import EdgeConfig, NodeConfig, StructureConfig
-from ..config.label_config_base import LabelConfig
-from ..config.feature_config_base import FeatureConfig
-from ..data_transformations.dist_feature_transformer import DistFeatureTransformer
-from ..data_transformations.dist_label_loader import DistLabelLoader, SplitRates, CustomSplit
-from ..data_transformations import s3_utils, spark_utils
+from graphstorm_processing.config.config_parser import EdgeConfig, NodeConfig, StructureConfig
```

Comment:
Why change it to use graphstorm_processing.config.xxx ?

Reply:
pylint was throwing errors about trying to import packages beyond the top level. This ensures the right thing happens regardless of how the package is imported